README.md

```diff
@@ -17,3 +17,9 @@ Da ich nicht alle Orte ausführlich durchlesen kann, ist jeder Pull Request Revi
 ## Maintainer gesucht
 
 Ich kann hier nicht über alle Let's Git Repositories alles überblicken, weswegen ich mich freuen würde wenn wir bis zu 5 Maintainer für dieses Projekt finden. Gerne kann dafür in einem Issue diskutiert werden.
+
+## Ort in die interaktive Karte aufnehmen
```

Comment:
Diese Position finde ich unglücklich gewählt. 
Besser gefallen würde mir die Anleitung unter dem Punkt "Einen neuen Eintrag hinzufügen" 
Eventuell ist es auch sinnvoll, die Anleitung als einen Unterpunkt von "Einen neuen Eintrag hinzufügen" zu sehen (also als H3), weil das ganze ja zusammen gehört.

Reply:
Dieser und der nächste Kommentar sind aus meiner Sicht widersprüchlich. Bitte genauer erklären.

Event. könnte man "Ort in die interaktive Karte aufnehmen" als eigenen Punkt behalten und unter  "Einen neuen Eintrag hinzufügen" einen Hinweis darauf geben, wie:
Um Deinen Ort leichter finden zu können, bietet es sich an, diesen in die interaktive Karte (siehe unten) aufzunehmen._

Reply:
Ich meinte hauptsächlich die Reihenfolge. 
Vom logischen Aufbau her würde es mir besser gefallen, wenn die Anleitung zur Karte direkt unter der Anleitung zu einem neuen Eintrag kommen würde. 

Das mit H3 als Unterpunkt ist mir nicht ganz so wichtig gewesen, würde aber verdeutlichen, dass der Karteneintrag Bestandteile eines neu eingetragenen Orts sein soll.